activerecord/lib/active_record/delegated_type.rb

```diff
@@ -136,6 +136,16 @@ module ActiveRecord
   #   end
   #
   # Now you can list a bunch of entries, call +Entry#title+, and polymorphism will provide you with the answer.
+  #
+  # == Active Record Nested Attributes
+  #
+  # Enabling nested attributes on a delegated_type association allows you to
+  # create the entry and message in one go:
+  #
```

Comment:
```suggestion
  #
  #    class Entry < ApplicationRecord
  #      delegated_type :entryable, types: %w[ Message Comment ]
  #      accepts_nested_attributes_for :entryable
  #    end
  #
```

The spacing might be a little off, but you get the idea. Let's add the example which includes `accepts_nested_attributes_for`